MANUAL_REGRESSION.md

```diff
@@ -190,32 +190,68 @@ Given webcam is not connected to the computer
 4. Open the cross device link on a mobile device that has media recorder API support (Chrome on Android)
    - user should be taken to the liveness intro screen
 
-### 10. Check happy path flow on other desktop browsers
+### 10a. Check happy path flow for passports on other desktop browsers
 
 (on private mode of: Safari, Firefox, IE11 and Microsoft Edge browsers)
 
 Go through the flow looking for layout/usability inconsistencies between browsers:
 
-1. Select one of the documents
+1. Select `passport` document
    - everything should be displayed properly and layout should not be broken
 2. Upload document
    - everything should be displayed properly and layout should not be broken
 3. Upload face photo
    - everything should be displayed properly and layout should not be broken
 
-### 11. Check happy path flow on mobile browsers
+### 10b. Check happy path flow for other document types on other desktop browsers
+
+(on private mode of: Safari, Firefox, IE11 and Microsoft Edge browsers)
+
+Go through the flow looking for layout/usability inconsistencies between browsers:
+
+1. Select either `driver's license` or `identity card` document
+   - everything should be displayed properly and layout should not be broken
+2. Type "de" in the text input below `Search for country`
+   - everything should be displayed properly and layout should not be broken
+   - there should be a dropdown list with at least 1 country displayed
```

Comment:
which country though should be displayed at least?

Reply:
Bangladesh, Russia Federation, Sweden seems to be the common countries between Driver's Licence and National ID. Can add it in. Was hesitant about it initially as was thinking this might change in future.

Reply:
its not a big deal however we could see the wrong results being displayed. Adding the name of the country or countries to be shown could be beneficial preventing a bug in the future and ensuring the quality is not lost

Reply:
Good point. Will add those in.